### PR TITLE
Updated Warnsveld & Added Fest Noz Hoorn

### PR DESCRIPTION
**Pull request by P-Storm**

### CODE REVIEW

File: events/netherlands/balfolk_nl.yaml [hunk 1]
```diff
@@ -91,6 +92,23 @@ events:
     social: true
     price: donation
 
+  - name: Fest Noz Hoorn
+    links:
+      - "https://www.balfolk.nl/agenda/fest-noz-hoorn/"
+      - "https://www.facebook.com/events/2864813930451126/"
```
Comment (qwandor) [inline events/netherlands/balfolk_nl.yaml:98]: This looks like the wrong Facebook event, it says "Balfolk Café Nijmegen met Hartwin Dhoore & Geronimo".

File: events/netherlands/balfolk_nl.yaml [hunk 2]
```diff
@@ -91,6 +92,23 @@ events:
...
+    country: Netherlands
+    city: Hoorn
+    styles: [balfolk]
+    workshop: true
+    social: true
+    bands:
+      - Beat Bouet Trio
+      - Fahrenheit
+      - La Sauterelle
+    price: €5-€15
```
Comment (qwandor) [inline events/netherlands/balfolk_nl.yaml:110]: This price doesn't match what the balfolk.nl page says.

File: events/netherlands/balfolk_nl.yaml (hunk 1 above)
Comment (P-Storm) [reply events/netherlands/balfolk_nl.yaml]: Was a copy paste error, took the format from another event, fixed in new commit

File: events/netherlands/balfolk_nl.yaml (hunk 2 above)
Comment (P-Storm) [reply events/netherlands/balfolk_nl.yaml]: Was a copy paste error, took the format from another event, fixed in new commit

**Comment by P-Storm** [comment]
Was a copy paste error, took the format from another event, fixed in new commit

File: events/netherlands/balfolk_nl.yaml (hunk 2 above)
Comment (P-Storm) [reply events/netherlands/balfolk_nl.yaml]: Without workshop it is 20 euro's, with workshop and fairtrade it is 45. Sorry for this error!

**Review by qwandor — APPROVED** [review]

**Pull request merged**